src/libs/ReportUtils.ts

```diff
@@ -4284,7 +4284,12 @@ function navigateToPrivateNotes(report: Report, session: Session) {
     Navigation.navigate(ROUTES.PRIVATE_NOTES_LIST.getRoute(report.reportID));
 }
 
+function getReportByID(reportID: string) {
+    return allReports ? allReports[reportID] : {};
+}
+
```

Comment:
is there a possibility of making this and `getReportActionsByReportID` functions easier to use by passing the ID without the Onyx key? IMO it's easier to use when you actually don't have to know the exact key